### ISSUE TITLE
Batch effect?

### ISSUE THREAD

**Issue by fpbarthel** [zd1/telseq#2]
Hey,

Do you have any idea if there could be batch effects when comparing data sequenced using different parameters on different platforms and/or aligned using different protocols.
1. Should there be batch effects comparing telseq results from lowpass sequencing to 30x depth WGS data? My impression is no, going by figure 2 from the supplements.
2. Should there be batch effects comparing telseq results from BAM files aligned using different protocols? My guess is probably not, but it seems hard to really asses this.

**Comment by zd1** [comment]
Hi there, for the first question, the coverage should be OK, but I am not sure about different platforms as I haven't tested that - the distribution of reads at GC% may vary depending on technology. The inter lane variation isn't very substantial in my experience. For the second question, alignment doesn't affect this as long as all reads are kept in the BAM.

**Comment by danielecook** [comment]
In terms of batch effects - I compared sequencing runs of the same DNA libraries of a set of strains of C. elegans. In one case the length of the reads differed (75bp; 100bp); however, the correlation was very high. I created a linear model and transformed the values.

I also examined depth of coverage because I had a large number of sequencing runs of the same DNA and again observed no correlation between depth of coverage and telomere length estimate. So it appears not to be an issue.

In a second case I compared strains prepared by different labs. We observed more variation but still a good correlation overall. Obviously - a lot of the variation there could be biological.
